docs/source/programming.rst

```diff
@@ -52,6 +52,8 @@ Additionally, developers can create an instance of a TornadoVM native array by i
    public static FloatArray fromSegment(MemorySegment segment);
    // from multiple TornadoVM native arrays to single TornadoVM native array
    public static FloatArray concat(FloatArray... arrays);
+   // from a slice of a TornadoVM native array
+   public static FloatArray slice(FloatArray array, int offset,int length);
```

Comment:
Update the documentation, as this is not static anymore

Reply:
thanks